### PR TITLE
Autotest 2726

### PR DESCRIPTION
**Pull request by monishaChengappa**

### CODE REVIEW

File: groupsList.php [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <?php
 class GroupsList {   
     private $groupRed = "";
-    private $groupGreen = "";
+    //private $groupGreen = "";
```
Comment (taoistmath) [inline groupsList.php]: Are we checking in a commented-out line?

File: js/controllers.js [hunk 2]
```diff
@@ -22,14 +22,16 @@ jehpControllers.controller('JobCtrl', function($scope, $http, $timeout, dataServ
     $scope.activeSubTab = 0;
     $scope.switchToSubTab = function(tabToSwitchTo){
       $scope.activeSubTab = tabToSwitchTo;
+      ;
```
Comment (taoistmath) [inline js/controllers.js]: is this semicolon needed?

File: js/controllers.js [hunk 3]
```diff
@@ -22,14 +22,16 @@ jehpControllers.controller('JobCtrl', function($scope, $http, $timeout, dataServ
...
     }
 
-
     $http.get('json/allJobs.json',
         //success
         function(data){
             $scope.jobs = data.jobs;
             $scope.groups = data.groups;
+            $scope.groupGreen = data.groupGreen;
+            $log.info("hi");
```
Comment (taoistmath) [inline js/controllers.js]: Can prob remove this log?

File: jehpDisplay.html [hunk 4]
```diff
@@ -34,40 +34,50 @@
...
+        <li class="passed">
+            <a href="" ng-click="toggle = !toggle">Passed</a>
+                <ul ng-show="toggle">  
+                  <li data-ng-repeat="grpGreen in groupGreen" id="{{grpGreen.name}}" ng-class="{active: $index == activeSubTab}">
+                    <a data-toggle="tab">
+                        {{grpGreen.name}} 
+                    </a>
+                  </li>
+               </ul>
+            </a>
```
Comment (taoistmath) [inline jehpDisplay.html]: Believe this is an extra </a> tag

**Comment by taoistmath** [comment]
+1

**Comment by dutchb** [comment]
:+1:

**Pull request merged**